prov/gni/src/gnix_ep.c

```diff
@@ -49,6 +52,142 @@ static LIST_HEAD(gnix_nic_list);
 static int gnix_ep_close(fid_t fid);
 static int gnix_ep_bind(fid_t fid, struct fid *bfid, uint64_t flags);
 
+
+/*
+ * function arrays to steer send/receive to ep type specific methods
+ */
+
+static ssize_t (* const send_method[])(struct fid_ep *ep,
```

Comment:
the style checker complained about defining typedefs.  maybe have to be in
include file.
On Apr 2, 2015 4:12 PM, "Ben Turrubiates" <notifications@github.com> wrote:

> In prov/gni/src/gnix_ep.c
> <https://github.com/ofi-cray/libfabric-cray/pull/63#discussion_r27702210>:
>
> > @@ -49,6 +52,142 @@ static LIST_HEAD(gnix_nic_list);
> >  static int gnix_ep_close(fid_t fid);
> >  static int gnix_ep_bind(fid_t fid, struct fid *bfid, uint64_t flags);
> >
> > +
> > +/*
> > + * function arrays to steer send/receive to ep type specific methods
> > + */
> > +
> > +static ssize_t (* const send_method[])(struct fid_ep *ep,
>
> @hppritcha <https://github.com/hppritcha> Doesn't like doing what?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ofi-cray/libfabric-cray/pull/63/files#r27702210>.
>